lib/Doctrine/ORM/EntityNotFoundException.php

```diff
@@ -30,8 +30,13 @@ class EntityNotFoundException extends ORMException
     /**
      * Constructor.
      */
-    public function __construct($class)
+    public function __construct($class, array $id)
```

Comment:
You should make `$id` optional otherwise it might break BC for people using this exception by hand...